### ISSUE TITLE
Failed to convert chapter to float

### ISSUE THREAD

**Issue by ANoobyDev** [mansuf/mangadex-downloader#7]
In some mangas, such as https://mangadex.org/title/3f28c47a-bf8d-4e79-83ca-2e64fe906372, the program fails to convert to float and ends prematurely due to a chapter being a string (in this case "EXTRA"). The error is on line 149 on the file "chapter.py" and the hack I made seems to fix it. 

```
try:
     num_chap = float(chapter.chapter)
except:
     print(f"{chapter.chapter}is not an integer, continue")
     break
```

**Comment by mansuf** [comment]
What chapter that triggered the error ?, and what language are you using ?

**Comment by mansuf** [comment]
Fixed in https://github.com/mansuf/mangadex-downloader/commit/2d4c6e94358efb05093ab48faa00cfc1113c98a1, thank you !

**Comment by mansuf** [comment]
Released in [`v0.5.0`](https://github.com/mansuf/mangadex-downloader/releases/tag/v0.5.0), you can start update mangadex-downloader now 👍

**Comment by ANoobyDev** [comment]
Nice
